universum.cabal

```diff
@@ -180,5 +185,7 @@ benchmark universum-benchmark
   if impl(ghc >= 8.10.1)
     ghc-options:         -Wno-missing-safe-haskell-mode
 
-  default-extensions:  NoImplicitPrelude
+  default-extensions:  ExplicitNamespaces
+                       NoImplicitPrelude
                        ScopedTypeVariables
+                       TypeOperators
```

Comment:
A couple of nitpicks:
1. According to [this page](https://ghc.gitlab.haskell.org/ghc/doc/users_guide/exts/explicit_namespaces.html), `ExplicitNamespaces` is implied by `TypeOperators`. Is it necessary to enable it explicitly?
2. Since `TypeOperators` is enabled for almost each component, it might be to put it into `common-options`.

Reply:
That makes sense, changed.